cache/filter.go

```diff
@@ -55,6 +55,40 @@ func LabelFilter(label string) Filter {
 	}
 }
 
+// ActorFilter return a Filter that match a bug actor
+func ActorFilter(actor string) Filter {
+	return func(repoCache *RepoCache, excerpt *BugExcerpt) bool {
+		for _, identityExcerpt := range repoCache.identitiesExcerpts {
```

Comment:
Two things:
1. you need to force `actor` to lower case to be fully case insensitive. Btw, let's call it `query`, that's what it is ;)
1. you are iterating over *all* the known identity in the repository, checking if they match the query and then checking if that's one of the actors of the bug. You can simply iterate over the actorIds of the bug, get the corresponding IdentityExcerpt (it's a map) and check if it match the query. Much less work to do !

Reply:
> You can simply iterate over the actorIds of the bug get the corresponding IdentityExcerpt

Yes totally agree. That's a better approach